### PR TITLE
Refactor Docker structure and update documentation

### PR DESCRIPTION
**Pull request by Ism1tha**
Introduce a new Docker structure with separate services for Apache and MariaDB, add phpMyAdmin, and update the README with installation instructions and application links. Clean up unused files and ensure proper configuration for the Docker environment.

### CODE REVIEW

**Review by albert1413 — APPROVED** [review]

File: .gitignore [hunk 1]
```diff
@@ -35,4 +35,8 @@
 .history
 .ionide
 
-# End of https://www.toptal.com/developers/gitignore/api/git,visualstudiocode
\ No newline at end of file
+# End of https://www.toptal.com/developers/gitignore/api/git,visualstudiocode
+
+# Ignore Composer files
+composer.lock
```
Comment (0x1026) [inline .gitignore:41]: Es recomanable fer commit del `composer.lock`: https://getcomposer.org/doc/01-basic-usage.md#commit-your-composer-lock-file-to-version-control
image

**Review by 0x1026 — CHANGES_REQUESTED** [review]
Molt bona feina, manquen un par de detallets ^^

File: composer.json [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+    "description": "Aplicatiu web per la gestió del manteniment d'arbrat urbà i periurbà.",
+    "type": "project",
+    "autoload": {
+        "psr-4": {
+            "App\\": "src/app"
+        }
+    },
+    "config": {
+        "vendor-dir": "src/vendor" 
+    },
```
[the remark this answers is not in the document]
Comment (Ism1tha) [reply composer.json:12]: No és possible, ja que les dependències han d'estar allà; si no, no podrem fer el require de l'autoload.

File: .gitignore (hunk 1 above)
Comment (Ism1tha) [reply .gitignore:41]: Vale.

**Review by 0x1026 — APPROVED** [review]

**Review by XavierChao — APPROVED** [review]

**Review by 0x1026 — APPROVED** [review]

**Review by martigarciall — APPROVED** [review]

**Review by Chuck-D-Norris — APPROVED** [review]

**Review by XavierChao — APPROVED** [review]

**Review by 0x1026 — APPROVED** [review]

**Review by SandraFerrando — APPROVED** [review]

**Pull request merged**